adflow/MExt.py

```diff
@@ -1,27 +1,28 @@
 import tempfile
-import imp
+import importlib
```

Comment:
When I try to test this locally, I run into this error: `AttributeError: module 'importlib' has no attribute 'util'`
Adding `import importlib.util` fixes the error. Did you run into this?

Reply:
No I didn't see this, very strange. What version of python are you running? Can you also check if `from importlib.util import find_spec` works? That may be more pythonic than your suggestion

Reply:
Yes, that also works. I'm running 3.8.5. The error does not show up when using `testflo` but does when running with `python`.

Reply:
Weird, I was testing some aerostructural scripts and didn't have any problems. I'm also on 3.8.5... anyways I will update the script. A quick google search shows that apparently importing the parent module does not automatically import sub-modules, so I guess in general it's always preferable to explicitly import the submodules that you want to use.